bloomfilter/Data/BloomFilter.hs

```diff
@@ -180,6 +183,15 @@ elemCheapHashes ch ub = go 0 where
                                then go (i + 1)
                                else False
 
+-- | Query several Bloom filters for membership of a single key. The result
+-- is equivalent to @map (elem k) bs@ but may be faster when used with many
+-- filters by taking advantage of shared calculations.
+--
+elemMany :: Hashable a => a -> Vec.Vector (Bloom a) -> UVec.Vector Bool
```

Comment:
doens't belong in the library. Why `Vec`, why not some other container? Oddly specific and ad-hoc.